gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -510,6 +504,9 @@ gst_tensor_time_sync_buffer_from_collectpad (GstCollectPads * collect,
     }
 
     if (!gst_tensor_buffer_append_memory (tensors_buf, mem, _info)) {
+      for (j = i + 1; j < counting; j++)
+        gst_memory_unref (in_mem[j]);
+
```

Comment:
I do not know well but,
Can this be solved with ```gst_buffer_unref(tensor_buf)``` ?

Reply:
The memories are not appended yet, so gst_buffer_unref() cannot release these.

Reply:
Thank you.